src/vs/workbench/contrib/chat/common/annotations.ts

```diff
@@ -61,6 +61,12 @@ export function annotateSpecialMarkdownContent(response: ReadonlyArray<IChatProg
 			} else {
 				result.push({ content: new MarkdownString(markdownText), kind: 'markdownContent' });
 			}
+		} else if (item.kind === 'codeblockUri') {
+			if (previousItem?.kind === 'markdownContent') {
+				const markdownText = `<vscode_codeblock_uri>${item.uri.toString()}</vscode_codeblock_uri>`;
```

Comment:
If we would ever finalize this api, or vulnerabilities, we'd want to think more about what happens if it's used incorrectly (outside of a codeblock)